content/client-server-api/modules/push.md

```diff
@@ -521,8 +521,43 @@ Definition:
 }
 ```
 
+**`.m.rule.is_user_mention`**
+
+Matches any message which contains the user's Matrix ID in the list of `user_ids`
+under the `m.mentions` property.
+
+Definition:
+
+```json
+{
+    "rule_id": ".m.rule.is_user_mention",
+    "default": true,
+    "enabled": true,
+    "conditions": [
+        {
+            "kind": "event_property_contains",
+            "key": "content.m\\.mentions.user_ids",
+            "value": "[the user's Matrix ID]"
+        }
+    ],
+    "actions": [
+        "notify",
+        {
+            "set_tweak": "sound",
+            "value": "default"
+        },
+        {
+            "set_tweak": "highlight"
+        }
+    ]
+}
+```
+
 **`.m.rule.contains_display_name`**
 
+As of `v1.7`, this rule is deprecated and should only be enabled if the event
+does not have an `m.mentions` property.
```

Comment:
> Hrm. As far as I know all the other push rule defintions contain their conditions in the json definition. Can we do that for this `m.mentions` condition?

There's no way to represent "if a property exists" in push rules, so this is done as an exception, not using a generic mechanism. [The MSC says](https://github.com/matrix-org/matrix-spec-proposals/blob/clokep/intentional-mentions/proposals/3952-intentional-mentions.md#backwards-compatibility):

> To avoid unintentional mentions these rules are modified to only apply when the `m.mentions` property is missing

Reply:
Hrm, that's awkward. I worry that this important piece of information is going to get overlooked. (Also, it's unsatisfactory that the JSON retrurned by `/pushrules` is no longer comprehensive, but the time to address that problem has passed).

Can we bold the text from "should only be enabled..." ?

Reply:
Also, link `m.mentions` back to `https://pr1508--matrix-spec-previews.netlify.app/client-server-api/#definition-mmentions` ?

Reply:
> it's unsatisfactory that the JSON retrurned by `/pushrules` is no longer comprehensive, but the time to address that problem has passed

I do hope that we can remove these rules fairly soon...